src/python_testing/TC_PWRTL_2_1.py

```diff
@@ -40,36 +40,43 @@ async def test_TC_PWRTL_2_1(self):
 
         attributes = Clusters.PowerTopology.Attributes
 
-        endpoint = self.user_params.get("endpoint", 1)
+        endpoint = 1
```

Comment:
Please leave as-is - testers need to be able to override this on the command line.

Reply:
Hi Cecille,
I have returned this to using the prior method for setting the endpoint variable.